common/class.Utils.php

```diff
@@ -181,6 +181,8 @@ public static function toHumanReadablePhpString($value, $indentation = 0)
                 }
             }
             $returnValue = "array(".substr($string, 0, -1).PHP_EOL.str_repeat('    ', $indentation).")";
+        } elseif (gettype($value) == "string") {
```

Comment:
It's better to use `is_string` function (and `is_array` above) since `gettype()` result strings are subject to change.